docs/src/main/asciidoc/datasource.adoc

```diff
@@ -155,29 +160,27 @@ quarkus.datasource.password=<your password>
 +
 You can also retrieve the password from Vault by link:{vault-datasource-guide}[using a credential provider] for your datasource.
 
-Until now, the configuration has been the same regardless of whether you’re using a JDBC or a reactive driver.
+Until now, the configuration has been the same regardless of whether you are using a JDBC or a reactive driver.
 When you have defined the database kind and the credentials, the rest depends on what type of driver you are using.
 It is possible to use JDBC and a reactive driver simultaneously.
 
 ==== JDBC datasource
 
 JDBC is the most common database connection pattern, typically needed when used in combination with non-reactive Hibernate ORM.
 
-. Add the `quarkus-agroal` dependency to your project:
+Using a built-in JDBC driver extension automatically includes the Agroal extension, which is the JDBC connection pool implementation applicable for custom and built-in JDBC drivers.
+However, for custom drivers, Agroal needs to be added explicitly.
+
+. To use a JDBC datasource, start with adding the necessary dependencies:
+
+..  For use with a custom JDBC driver, add the `quarkus-agroal` dependency to your project alongside the extension for your relational database driver:
 +
 [source,bash]
 ----
 ./mvnw quarkus:add-extension -Dextensions="agroal"
 ----
-+
-[TIP]
-====
-Agroal serves as the JDBC connection pool implementation and can be utilized by both custom and built-in JDBC drivers.
-Using a built-in JDBC driver extension automatically includes the Agroal extension.
-For custom drivers, Agroal needs to be added explicitly.
-====
 
-. Choose and add the Quarkus extension for your relational database driver from the list below:
+.. For use with a built-in JDBC driver, choose and add the Quarkus extension for your relational database driver from the list below:
```

Comment:
I'd put this first, since this is the most common case.

Reply:
Sure, will do.